examples/piecestore/rpc/client/api/api.go

```diff
@@ -0,0 +1,109 @@
+package api
+
+import (
+  "io"
+  "log"
+
+  "golang.org/x/net/context"
+
+  "google.golang.org/grpc"
+
+  "github.com/zeebo/errs"
+
+  pb "storj.io/storj/examples/piecestore/rpc/protobuf"
+)
+
+var ServerError = errs.Class("serverError")
+
+// Struct containing Shard information from ShardMetaRequest
+type ShardMeta struct {
```

Comment:
If this is a _piece_ storage service, shouldn't it call it `Piece` instead of `Shard`?